Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -395,8 +395,8 @@ - (void)launchApplicationWithContext:(BPExecutionContext *)context {
         NEXT([__self deleteSimulatorWithContext:context andStatus:BPExitStatusLaunchAppFailed]);
     };
 
-    [context.runner launchApplicationAndExecuteTestsWithParser:context.parser andCompletion:handler.defaultHandlerBlock isHostApp:NO];
-    [BPUtils printInfo:INFO withString:[@"Yay, after the launch async call!!!" stringByAppendingString:stepName]];
+    [context.runner launchApplicationAndExecuteTestsWithParser:context.parser andCompletion:handler.defaultHandlerBlock];
+    [BPUtils printInfo:INFO withString:[@"Async Launch Returned" stringByAppendingString:stepName]];
```

Comment:
I think since the particular problem I was trying to detect the existence of didn't actually exist, this can probably go. While it's nice to know than an async method finished, it seems like if we give this one special treatment, they're all going to start demanding it. :)